cluster/addons/dns/kube-dns.yaml.base

```diff
@@ -84,6 +84,7 @@ spec:
       annotations:
         scheduler.alpha.kubernetes.io/critical-pod: ''
     spec:
+      priorityClassName: system-cluster-critical
       tolerations:
       - key: "CriticalAddonsOnly"
```

Comment:
Do we still need these tolerations or can we have a comment?

Reply:
We should leave these for now. These shouldn't hurt and in case we run into any issues and are forced to disable priority and preemption, we won't need to add these again.